widgetssdk/src/main/AndroidManifest.xml

```diff
@@ -29,7 +29,7 @@
         <activity
             android:name=".survey.SurveyActivity"
             android:launchMode="singleTop"
-            android:theme="@style/Application.Glia.Chat.Activity" />
+            android:theme="@style/Application.Glia.Call.Activity" />
```

Comment:
Maybe rename the theme since it's not CallActivity's theme anymore, but is used by multiple activities now.